docs/data-model.asciidoc

```diff
@@ -337,6 +337,34 @@ You can filter and group by these dimensions (some of which are optional, for ex
 
 The `@timestamp` field of these documents holds the start of the aggregation interval.
 
+[float]
+===== Service-transaction metrics
+
+To power {kibana-ref}/xpack-apm.html[{apm-app}] visualizations,
+APM Server aggregates transaction events into service-transaction metrics.
+
+*`transaction.duration.summary`* and *`transaction.duration.histogram`*::
++
+--
+These metrics measure the aggregated latency and latency distribution of service transaction groups,
+used to power visualizations and analytics in Elastic APM.
```

Comment:
Do we expect users to be able to know the difference between transaction groups and service transaction groups?

Reply:
Actually how do we explain the service transaction grouping?

Reply:
How about we highlight that they are generally producing the same kind of metrics, but that the service transaction metrics usually have a much lower cardinality as they have significantly less dimensions. The UI uses them where less details of the transactions are needed - so basically service transaction metrics are used for service oriented visualisations, whereas transaction metrics are powerin transaction oriented pages, where more details and slicing of the transaction groups is helpful.